apps/bytebank-web/app/api/transactions/[id]/route.ts

```diff
@@ -15,3 +15,46 @@ export async function GET(req: NextRequest, { params }: { params: Promise<RouteP
   const data = await response.json();
   return Response.json(data);
 }
+
+export async function UPDATE(req: NextRequest, { params }: { params: Promise<RouteParams> }) {
```

Comment:
Update não é uma rota válida no next js, precisa usar put.

```suggestion
export async function PUT(req: NextRequest, { params }: { params: Promise<RouteParams> }) {
```